backend/apps/core/tests/test_throttling.py

```
@@ -0,0 +1,238 @@
+        # First request should be allowed
+        self.assertTrue(allowed)
+        self.assertIsNone(wait_time)
+
+
+class RateLimitResponseTests(ThrottlingTestCase):
+    """Test rate limit response format."""
+
+    def test_rate_limit_response_format(self):
+        """Test that rate limit responses have correct format."""
+        # Make enough requests to hit rate limit
+        for i in range(101):
```

Ruff (0.14.3)

Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)